src/components/sbb-toggle-check/sbb-toggle-check.scss

```diff
@@ -0,0 +1,92 @@
+.toggle-check {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  color: var(--toggle-check-color);
+  @include text-s--regular;
+}
+
+.toggle-check__slider--after {
+  margin-inline-end: var(--sbb-spacing-fixed-3x);
+}
+
+.toggle-check__slider--before {
+  margin-inline-start: var(--sbb-spacing-fixed-3x);
+}
+
+input[type='checkbox'] {
+  position: absolute;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  margin: 0;
+  width: 100%;
+  height: 100%;
+  cursor: inherit;
+  z-index: -1;
```

Comment:
I would recommend also adding the comment, as it describes the purpose of the z-index